src/styles/2-components/_calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $ceil-size-x * 7 + 14px;
+  padding: 10px;
+  gap: 1px;
+
+  &__day {
+    width: $ceil-size-x;
+    height: $ceil-size-y;
+    background-color: $grey;
+    margin-left: $current-day;
+    color: $black;
+    box-sizing: border-box;
```

Comment:
Better reset border sizing along with margin

https://css-tricks.com/box-sizing/#aa-good-better-and-probably-best-box-sizing-reset-methods

Reply:
Misha said that you don't need to use the asterisk selector so I was a little bit confused but thanks for this information